hubblestack/daemon.py

```diff
@@ -788,6 +790,7 @@ def refresh_grains(initial=False):
 
     hubblestack.utils.stdrec.__grains__ = __grains__
     hubblestack.utils.stdrec.__opts__ = __opts__
+    hubblestack.utils.osquery_lib.__grains__ = __grains__
```

Comment:
Can we just add `global __grains__` to the util module?

Reply:
Removed it from here. But actually we can't use `__grains__` in util, since the util is imported in docker grain and the variable `__grains__` is only initialised after loading grains.